api/database_cluster_restore.go

```diff
@@ -79,33 +79,33 @@ func (e *EverestServer) CreateDatabaseClusterRestore(ctx echo.Context) error {
 		})
 	}
 
-	return e.proxyKubernetes(ctx, "", databaseClusterRestoreKind, "")
+	return e.proxyKubernetes(ctx, namespace, databaseClusterRestoreKind, "")
 }
 
 // DeleteDatabaseClusterRestore Delete the specified cluster restore on the specified kubernetes cluster.
-func (e *EverestServer) DeleteDatabaseClusterRestore(ctx echo.Context, name string) error {
-	return e.proxyKubernetes(ctx, "", databaseClusterRestoreKind, name)
+func (e *EverestServer) DeleteDatabaseClusterRestore(ctx echo.Context, namespace string, name string) error {
```

Comment:
```suggestion
func (e *EverestServer) DeleteDatabaseClusterRestore(ctx echo.Context, namespace, name string) error {
```
There are many of such in the PR, not a big deal, it's just triggering my OCD😅

Reply:
Fixed :stuck_out_tongue: I can't fix the autogenerated code but I think everything else got fixed.